handlers/main.yml

```diff
@@ -24,9 +24,7 @@
     state: stopped
 
 - name: Sync Keystone database
-  command: keystone-manage db_sync
-  become: True
-  become_user: keystone
```

Comment:
The workaround for this would be use Ansible 2.1 with the `allow_world_readable_tmpfiles` option enable.
We can change the requirements doc (is the latest stable version of Ansible)